keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/CombinedRepository.kt

```diff
@@ -313,20 +312,14 @@ class CombinedRepository(
   override fun getAllArtifacts(type: ArtifactType?): List<DeliveryArtifact> =
     artifactRepository.getAll(type)
 
-  override fun storeArtifact(name: String, type: ArtifactType, version: String, status: ArtifactStatus?): Boolean =
-    artifactRepository.store(name, type, version, status)
+  override fun storeArtifactVersion(artifactVersion: PublishedArtifact): Boolean =
+    artifactRepository.storeArtifactVersion(artifactVersion)
 
-  override fun storeArtifact(artifact: DeliveryArtifact, version: String, status: ArtifactStatus?): Boolean =
-    artifactRepository.store(artifact, version, status)
+  override fun getArtifactVersion(name: String, type: ArtifactType, version: String, status: ArtifactStatus?): PublishedArtifact? =
```

Comment:
the name `getArtifactVersion` is a bit confusing to me, as it's indicated we are getting the artifact version and not `PublishedArtifact` object -- should we maybe remane it to `getArtifact` or `getArtifactByxxx`?

Reply:
Yeah, I know... We've been having a conversation about the naming of the classes used to represent artifacts. I think the last proposal we landed on would be to rename `DeliveryArtifact` to `ArtifactSpec` and `PublishedArtifact` to just `Artifact`, when it would then make total sense to rename these methods. I'd like to tackle that in a separate PR though, if that's OK.

Reply:
Cc: @robfletcher

Reply:
@gal-yardeni Update: I went ahead and renamed the methods per your suggestion.